renderer-macos/lib/stubs/BriskCoreFoundation.c

```diff
@@ -0,0 +1,17 @@
+#import "BriskCocoa.h"
+#import <ApplicationServices/ApplicationServices.h>
+#import <CoreFoundation/CFBundle.h>
+
+void ml_BriskCoreFoundation_openURL(value url_v) {
+  CAMLparam1(url_v);
+
+  char *utf8string = String_val(url_v);
+  CFURLRef url = CFURLCreateWithBytes(NULL,                   // allocator
```

Comment:
We might eventually want to switch to ocaml-osx-cf for `CF` bindings.

Reply:
<img width="723" alt="image" src="https://user-images.githubusercontent.com/1693190/55286805-e302e000-53a0-11e9-9090-0d54aeec23ca.png">

Reply:
Yeah, we'll have to write `CFURL` ourselves. 😁